hack/tools/release/notes.go

```diff
@@ -90,8 +93,15 @@ func firstCommit() string {
 }
 
 func run() int {
-	lastTag := lastTag()
-	cmd := exec.Command("git", "rev-list", lastTag+"..HEAD", "--merges", "--pretty=format:%B") //nolint:gosec
+	var lastTag string
```

Comment:
Can we add an additional new variable for the range. Just feels a bit hacky to write since-until into "lastTag"

Reply:
I renamed the variable to describe the content since it's used later in the script and avoid more if this else that.